documentation/htdocs/index.html

```diff
@@ -57,12 +57,18 @@ <h2>Features</h2>
    <li>Highly configurable - 608 configurable options as of version 0.66</li>
 </ul>
 
+<a href="uncrustify.html">What it does?</a>
+
 <p>
```

Comment:
I wish that the related file to be as current as possible, in the same PR

Reply:
This was more to show unlinked files (maybe just to delete them) 

https://github.com/uncrustify/uncrustify/blob/master/documentation/htdocs/uncrustify.html
> I'm not sure if I can describe all things uncrustify does at this time. Let me give it a try!

Then I will try to link other txt files there too

Reply:
I wait until you are ready

Reply:
I really try, but don't know what uncrustify support or not at this time:

This is the content of that file (esentially):

What it does:

        - Indent code
	- Align code on '=' and variable definitions
	- Align #define stuff
	- Align backslash-newline stuff
	- Reformat comments (a little bit)
	- Fix inter-character spacing

What it doesn't do (yet):
	- Split long lines
	- Handle format changing #if/#else stuff

Please provide feedback on this

Reply:
Thanks for linking the two files, I looked on both of the and they are really outdated (uncrustify.html) or not exactly something that I would want to link right now (options.html) 

I am not entirely sure what you are doing now but please don't try to describe all options on you own. That is a way too big target that should be done gradually.

I think we should discuss that again in #1562 and close this PR.